controller/src/main/java/org/jboss/as/controller/AbstractAttributeDefinitionBuilder.java

```diff
@@ -414,6 +414,16 @@ public BUILDER setStorageRuntime() {
     }
 
     /**
+     * Adds the {@link AttributeAccess.Flag#STORAGE_RUNTIME} flag and removes any conflicting flag.
+     *
+     * @return a builder that can be used to continue building the attribute definition
+     */
+    public BUILDER setStorageRuntime(boolean forceRegistration) {
```

Comment:
I removed this `setStorageRuntime(boolean)` method and added a `forceRegistration()`